cluster/gce/configure-vm.sh

```diff
@@ -890,6 +890,45 @@ contexts:
 EOF
   fi
 
+
+  if [[ -n "${GCP_IMAGE_VERIFICATION_URL:-}" ]]; then
+    # This is the config file for the image review webhook.
+    cat <<EOF >>/etc/salt/minion.d/grains.conf
+  image_review_config: /etc/gcp_image_review.config
+EOF
+    cat <<EOF >/etc/gcp_image_review.config
+clusters:
+  - name: gcp-image-review-server
+    cluster:
+      server: ${GCP_IMAGE_VERIFICATION_URL}
+users:
+  - name: kube-apiserver
+    user:
+      auth-provider:
+        name: gcp
+current-context: webhook
+contexts:
+- context:
+    cluster: gcp-image-review-server
+    user: kube-apiserver
+  name: webhook
+EOF
+    # This is the config for the image review admission controller.
+    cat <<EOF >>/etc/salt/minion.d/grains.conf
+  image_review_webhook_config: /etc/image_review_admission_controller.config
+EOF
+    cat <<EOF >/etc/image_review_admission_controller.config
+ImagePolicy:
```

Comment:
@ecordell Ok, that was the part I was the most unclear about. Does it look right now? Additionally, I suppose I should only include the --admission-control-config-file argument if ImagePolicyWebhook is set?